docs/access-api-keys.asciidoc

```diff
@@ -123,11 +123,15 @@ POST /_security/api_key
         {
           "names": ["{beat_default_index_prefix}-*"],
           "privileges": ["create_index", "create_doc"]
-        },
+        }
+      ]
+    },
+    "{beat_default_index_prefix}_sourcemap": {
```

Comment:
tbh I'm not sure what's the point of this example, since the permissions are good enough to run apm-server with basic functionalities + rum but it is still lacking agentcfg role and monitoring role etc. I've updated this section so that it follows what's recommended in the updated RUM role. I'm wondering if we should expand this example to include all roles that allow apm-server to do everything it'll do but potentially making the doc hard to maintain, or stick with what we have now. In any case, this shouldn't block this PR.

Reply:
This example is about "Create an API key for writing events" and RUM source mapping kind of falls under it but at the same time not exactly.

Reply:
This is a good point. I think adding agent config makes sense. We should add it to both of the examples on this page (like you did with source maps). Monitoring I think we're okay to leave separate. It already has a separate section on this page.

Reply:
Effectively the doc compiles to `.apm-source-map`, but I agree that it is better to hardcode it.

Reply:
added agentcfg role to example.